build/nVentive-generic.ruleset

```diff
@@ -7,13 +7,11 @@
     <Rule Id="CA1021" Action="Warning" />
     <Rule Id="CA1033" Action="Warning" />
     <Rule Id="CA1038" Action="Warning" />
-    <Rule Id="CA1044" Action="Error" />
```

Comment:
These ruleset files are not actually used. But before deleting them, I want to match their intent as much as possible.